packages/models-library/tests/test_utils_database_models_factory.py

```diff
@@ -19,18 +21,22 @@
 @pytest.mark.parametrize("table_cls", metadata.tables.values(), ids=lambda t: t.name)
 def test_table_to_pydantic_models(table_cls):
 
-    PydanticModelAtDB = sa_table_to_pydantic_model(table=table_cls)
-    assert issubclass(PydanticModelAtDB, BaseModel)
+    PydanticOrm = create_pydantic_model_from_sa_table(
```

Comment:
I mean to express that it is a model from the database. was taken from https://pydantic-docs.helpmanual.io/usage/models/#orm-mode-aka-arbitrary-class-instances
Strictly speaking it might not be an ORM
https://en.wikipedia.org/wiki/Object%E2%80%93relational_mapping

Reply:
It's "data to object  mapper" without the persistence part. I think ORMs also handle syncing data to and from the DB.

Reply:
Some ORMs use the term "mapped class". Otherwise ORM is fine but a bit misleading (at least for me). If it's called ORM I expect it to have DB interaction.

Reply:
will change in the next PR